girder/models/model_base.py

```diff
@@ -735,17 +733,17 @@ def _removeSupplementalFields(doc, fields):
 
         whitelist = []
         if isinstance(fields, dict):
-            for k, v in six.viewitems(fields):
+            for k, v in fields.items():
                 if not v:
                     doc.pop(k, None)
                 else:
                     whitelist.append(k)
             if whitelist:
-                for k in list(six.viewkeys(doc)):
+                for k in list(doc.keys()):
```

Comment:
I'd prefer these as dict comprehensions but don't worry about it if you don't feel like it.

Reply:
Nevermind, I just realized this has to operate on the input rather than return a new dict. Womp womp.